cmd/ferretdb/main.go

```diff
@@ -70,15 +73,20 @@ func initFlags() {
 	handlers := maps.Keys(registeredHandlers)
 	slices.Sort(handlers)
 	flag.Lookup("handler").Usage = "backend handler: " + strings.Join(handlers, ", ")
+	flag.Lookup("log-level").Usage = "sets logging levels"
```

Comment:
The idea was that we could use code to set the help text and the default value, something that is not possible in a variable declaration. In pseudocode:

```go
var levels []string
for _, l := range []{zap.Debug, ...} {
  levels = append(levels, l.String())
}
flag.Lookup("log-level").Usage = "log level: " + strings.Join(levels, ", ")
```